### PR TITLE
Helm Chart updates

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: GitHub <noreply@github.com>

### CODE REVIEW

File: helm-chart/templates/nextcloud-aio-talk-service.yaml [hunk 1]
```diff
@@ -17,6 +18,9 @@ spec:
       port: {{ .Values.TALK_PORT }}
       protocol: UDP
       targetPort: {{ .Values.TALK_PORT }}
+    - name: "8081"
+      port: 8081
+      targetPort: 8081
```
Comment (szaimen) [inline helm-chart/templates/nextcloud-aio-talk-service.yaml:23]: this port does not need to be published. How can I let it only work internally?

Comment (st3iny) [reply helm-chart/templates/nextcloud-aio-talk-service.yaml:23]: The default type is `ClusterIP` and it will only be reachable from inside the cluster.

Comment (szaimen) [reply helm-chart/templates/nextcloud-aio-talk-service.yaml:23]: okay good! Thanks!

Comment (szaimen) [reply helm-chart/templates/nextcloud-aio-talk-service.yaml:23]: I suppose I might need to add another service file manuay that includes this port?

Comment (szaimen) [reply helm-chart/templates/nextcloud-aio-talk-service.yaml:23]: Or can I limit the type somehow to specific ports?

Comment (st3iny) [reply helm-chart/templates/nextcloud-aio-talk-service.yaml:23]: You need to split the internal ports to another service. There is no way to selectively publish ports from a single service.

Comment (szaimen) [reply helm-chart/templates/nextcloud-aio-talk-service.yaml:23]: okay, I thought so. Thanks! :)

Comment (szaimen) [reply helm-chart/templates/nextcloud-aio-talk-service.yaml:23]: will do in a follow-up.

**Pull request closed without merging**